tools/c7n_index/setup.py

```diff
@@ -28,5 +28,5 @@
     entry_points={
         'console_scripts': [
             'c7n-indexer = c7n_index.metrics:cli']},
-    install_requires=["c7n", "click", "influxdb", "elasticsearch"],
+    install_requires=["c7n", "click", "influxdb", "elasticsearch", "elasticmock"],
```

Comment:
`elasticmock` only required for testing?

Reply:
Yes, it is only required for testing.

Reply:
thats typically a test_requires .. install requires means anyone using, not testing, it really should just be in a test-requirements.txt then.

Reply:
I understand. but if we use a txt file to install req, it would have to be specified in a Readme(to pip install) or this can be added to setup.py as `tests_require = ['elasticmock']`. For simplicity, should I consider the second option?